scrapy/http/cookies.py

```diff
@@ -58,8 +58,8 @@ def _cookies(self):
     def clear_session_cookies(self, *args, **kwargs):
         return self.jar.clear_session_cookies(*args, **kwargs)
 
-    def clear(self):
-        return self.jar.clear()
+    def clear(self, domain=None, path=None, name=None):
```

Comment:
What do you think about using args/kwargs, like in `clear_session_cookies`? This way we won't have to change this method if Python decide to change signature for some reason.

Reply:
Currently there're two types of function signatures here:
- Using args/kwargs, like `clear_session_cookies`.
- Using the exact signature of the underlying CookieJar object, like `set_cookie` and `set_policy`.

For now I don't think any type has a significant advantage to the other. Thus I'm okay to use either type :) Please let me know if you want me to make the change.

Reply:
Fair enough, I'm fine with merging it as-is.